superset/views.py

```diff
@@ -1484,6 +1486,13 @@ def explore(self, datasource_type, datasource_id):
             return self.save_or_overwrite_slice(
                 request.args, slc, slice_add_perm, slice_edit_perm)
 
+        # find out if user is in explore v2 beta group
+        # and set flag `is_in_explore_v2_beta`
+        is_in_explore_v2_beta = False
+        roles = (r.name for r in get_user_roles())
```

Comment:
s/roles = (r.name for r in get_user_roles())/roles = [r.name for r in get_user_roles()]
using list will be safer here